analyzers/FacsCalibur/nbproject/build-impl.xml

```diff
@@ -0,0 +1,1413 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file should not be added to version control. It belongs with the IDE (Netbeans)